sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -9,6 +9,8 @@
 - Removed types that were accidentally exposed from two modules. Only `BlobServiceClient`, `ContainerClient`, 
 `BlobClient` and `LeaseClient` should be imported from azure.storage.blob.aio
 - `Logging` has been renamed to `BlobAnalyticsLogging`.
+- Client and model files have been renamed to be internal. Users should import from the top level modules: 
```

Comment:
how about have been "made" internal?

Reply:
I'll rephrase. This change was made in files and queues as well.